test/test_smoke.jl

```diff
@@ -16,7 +16,7 @@ Aqua.test_all(
     undefined_exports = false,
     project_extras = false,
     stale_deps = false,
-    deps_compat = false,
+    deps_compat = (; check_extras = true, check_weakdeps = true),
```

Comment:
Those tests are empty for Aqua itself, which probably explains why the corresponding code lines are not hit

Reply:
The code does only detect that there are not extras and weakdeps way deeper in the code. And it's only the `@testset` lines that are not hit, everything inside is hit. So this is just an example of bad coverage detection